server/resource_server/service/service.go

```diff
@@ -165,7 +165,7 @@ func (s *service) ProfileClient(userName string) (models.ClientResponseOutput, e
 	clientModel := models.ClientResponseOutput{
 		ID:          clientData.ID,
 		Secret:      clientData.Secret,
-		Name:        clientData.Username,
+		Name:        clientData.Name,
```

Comment:
Aren't there any implications with this? Maybe parts of the server that currently expect username from here?

Reply:
Frontend is waiting Name, that's why I changed Username to Name, I checked and it is working now.